changes/1207.breaking.md

```diff
@@ -0,0 +1,2 @@
+`Member.mention` now returns a mention string without the `!` bang regardless of nickname setting.
+ - Example: `<@1234567890>` instead of `<@!1234567890>`
```

Comment:
This is not really necessary

And would this be considered a breaking change even?

Reply:
I am not sure exactly about this. I thought this was old behavior by Discord that should not be there anymore. I was encountering some issues pertaining to this and noticed this here in hikari. 

> And would this be considered a breaking change even?

It modifies the return value of the mention by removing the bang. If this was implemented, it could be a breaking change as it may perhaps change the behavior of comparisons. I don't know how different clients behave...

Guess this isn't needed to be modified.

Reply:
Resolved in Discord